### DISCUSSION TITLE
Configure Fresh Install to use 443

### DISCUSSION THREAD

**Discussion by Alex-Giaquinto** [ToolJet/ToolJet#9209] [Q&A]
Hello! Just reaching out to get some support on what should be done to my fresh install in order to make sure the client is using 443. I am sure I need to make changes to both the docker-compose file and my .env file, but just wanted to ask here to see if I can save myself some time. Thank you!

**Comment by Navaneeth-pk** [comment]
Hello,

It will be easier to provide support via our community in this specific case. Here is the invite to join: https://tooljet.com/slack.
